infra/templates/az-isolated-service-single-region/app.tf

```diff
@@ -33,3 +33,76 @@ module "keyvault" {
     "azurerm" = "azurerm.app_dev"
   }
 }
+
+module "container_registry" {
+  source                  = "../../modules/providers/azure/container-registry"
+  container_registry_name = local.acr_name
+  resource_group_name     = azurerm_resource_group.app_rg.name
+  // Note: this is requird until App Services and ACR work over MSI. See the design document for more details.
+  container_registry_admin_enabled = true
```

Comment:
This should be driven from a template variable as the value of admin_enabled will be changing soon.

Reply:
We don't support other use cases at the moment so if this is not `true` then it will result in a failure. 

Options like this seem useful for modules because they make them flexible, but not as useful for the templates which have a more opinionated stance on the configuration.